php/Terminus/Models/Environment.php

```diff
@@ -876,6 +869,63 @@ public function workflow($workflow) {
   }
 
   /**
+   * Sifts through the backups data and adds needful properties
+   *
+   * @param [stdClass] $raw_backups Backups data object from Request data
+   * @return [array] $backups
+   */
+  private function annotateBackups($raw_backups) {
```

Comment:
I was holding off on implementing the Backups model/collection until we got the go-ahead to create BackupsCommand, but perhaps that's not necessary. I could move it now.